### ISSUE TITLE
Glew library not generated

### ISSUE THREAD

**Issue by hadialhussieni** [digipen-team-uwu/uwu-engine#11]
Linker error
```
C:\Users\hadia\Desktop\team3\umbra\ninja : warning : multiple rules generate lib/glewd.lib. builds involving this target will not be correct; continuing anyway [-w dupbuild=warn]
C:\Users\hadia\Desktop\team3\umbra\ninja : warning : multiple rules generate lib/glewmxd.lib. builds involving this target will not be correct; continuing anyway [-w dupbuild=warn]
```

**Comment by zachrammell** [comment]
not reproducible on Debian Bullseye with gcc
or Pop!_OS with gcc

**Comment by chaun-nguyen** [comment]
> Linker error
> 
> ```
> C:\Users\hadia\Desktop\team3\umbra\ninja : warning : multiple rules generate lib/glewd.lib. builds involving this target will not be correct; continuing anyway [-w dupbuild=warn]
> C:\Users\hadia\Desktop\team3\umbra\ninja : warning : multiple rules generate lib/glewmxd.lib. builds involving this target will not be correct; continuing anyway [-w dupbuild=warn]
> ```

It happens to me too :(

**Comment by HardGaming01** [comment]
This is caused by glew not having a way to rebuild itself. What rebuild does is it clears "Build" cache and build again and this clear also clears the exported glew.
![image](https://user-images.githubusercontent.com/11523007/72003270-d21fae80-31fd-11ea-9d6f-66d74fdce7d6.png)

What we want to do is to clear the CMake cache
![image](https://user-images.githubusercontent.com/11523007/72003343-01ceb680-31fe-11ea-985d-d0b1d1fbdc41.png)

This is a temporary workaround.
There are 3 possible solution to this is: 

1. change glew's cmake file to manage it's build process by ourselves.
2. Link glew as pre-built binary (the same way we link FMOD). we'll not be including it as sub module in this way.
3. use another openGL function loader

**Comment by zachrammell** [comment]
fixed by 55d4a9be9eb32b9bd45b803cb73ff71def2a2d91
